src/components/EnvelopeSkeleton.vue

```diff
@@ -552,24 +552,51 @@ export default {
 		--list-item-border-radius: var(--border-radius-element, calc(var(--default-clickable-area) / 2));
 		padding-block: calc(var(--list-item-padding) * 2);
 		--list-item-padding: 2px;
+		height: unset;
 
-		.list-item-content__inner__main {
-			display: flex;
-			justify-content: start;
-			gap: 12px;
-			min-width: 0;
-		}
-		.list-item-content__inner__details {
+		.list-item-content {
 			flex-direction: row;
-			align-items: unset;
-			justify-content: end;
-		}
-		.list-item-content__name {
-			align-self: center;
+			align-content: center;
+			align-items: center;
+
+			&__name {
+				align-self: center;
+				min-width: 300px;
+				padding-right: calc(var(--default-grid-baseline) * 2);
+			}
+
+			&__inner {
+				overflow-y: hidden;
+			}
+
+			&__inner__main {
+				display: flex;
+				justify-content: start;
+				min-width: 0;
+			}
+
+			&__inner__details {
+				flex-direction: row;
+				align-items: unset;
+				justify-content: end;
+				margin-top: 0;
+				margin-left: 0;
+			}
 		}
 
 		a {
 			margin: 0;
+			align-items: center;
+			height: unset;
+		}
+
+		:deep(.avatardiv__initials) {
+			line-height: 40px !important;
```

Comment:
why the deep overwrite?

Reply:
Double-checked and it is indeed not needed. The idea was to have the avatar be `40px` in oneline and `45px` (as per Marco's suggestion some time ago), but now I noticed that Greta made it `40px` everywhere and not just in oneline, I assume per a new design decision.

Reply:
yes, the avatar should be 40px everywhere on mail :)